src/rest_api/classes/antibody/widgets/overview.clj

```diff
@@ -0,0 +1,45 @@
+(ns rest-api.classes.antibody.widgets.overview
+  (:require
+   [clojure.string :as str]
+   [rest-api.classes.generic-fields :as generic]
+   [rest-api.formatters.object :as obj :refer [pack-obj]]))
+
+(defn corresponding-gene [antibody]
+  {:data (if-let [holders (:antibody/gene antibody)]
+           (pack-obj (:antibody.gene/gene (first holders))))
+   :description "the corresponding gene the antibody was generated against"})
+
+(defn antigen [antibody]
+  {:data {:comment (if-let [antigen (:antibody/antigen antibody)]
+                      (:antibody.antigen/text antigen))
+          :type (if-let [antigen (:antibody/antigen antibody)]
+                    (str/capitalize (name (:antibody.antigen/value antigen))))}
+   :description "the type and decsription of antigen this antibody was generated against"})
+
+(defn constructed-by [antibody] ; no examples in datomic
```

Comment:
Just wondering - is the comment something that needs fixing/a TODO or just an observation?